osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -64,6 +64,16 @@ public class BeatmapCarousel : CompositeDrawable, IKeyBindingHandler<GlobalActio
 
         private CarouselBeatmapSet selectedBeatmapSet;
 
+        /// <summary>
+        /// Beatmapsets that were selected but got filtered. Used to try and re-select if only filter actions have been applied.
```

Comment:
you mean if explicit selection hasn't taken place? that sounds like a better way to explain it